src/core/algorithms/ind/ind.h

```diff
@@ -4,6 +4,7 @@
 #include <string>
 
 #include "model/table/column_combination.h"
+#include "model/table/relational_schema.h"
```

Comment:
Add include for `Vertical`, because `RelationalSchema` uses forward declaration:
```cpp
#inlcude "model/table/vertical.h"
```